### ISSUE TITLE
Tooltip is not available on toolbar buttons for the keyboard users

### ISSUE THREAD

**Issue by sryarama** [ckeditor/ckeditor4#4324]
Any content that can be surfaced by a mouse user should be surfaced when in a keyboard-only scenario (not just screen reader users). Screen reader users aren't the only scenario for keyboard-only scenarios.

Using "title" property on an element uses the browser's default tooltip. However, because this will never surface to keyboard users, this is not considered an accessible approach. 

As per our accessibility testing team, element's title property is not accessible to keyboard users which is an issue.  FYI...One our A11y SME has a whole article discussing the history and recommended approaches: https://sarahmhigley.com/writing/tooltips-in-wcag-21/

**Comment by Dumluregn** [comment]
It's a duplicate of #2613.

**Comment by VienTranSB** [comment]
ADADSADSAD
